README.md

```diff
@@ -46,31 +58,18 @@ Open the file `carto-values.yaml` that you have received and configure the neede
 - Configure [external DBs](#external-databases)
 - Configure external buckets
 
-4. Deploy CARTO
-
-```bash
-helm install carto-selfhosted-v1 carto-selfhosted-charts/carto -f carto-values.yaml -f carto-secrets.yaml
-```
-
-5. Configure your DNS to point to the deployment:
-
-  In GKE:
-
-  Create an A record pointing to the IP:
-
-  ```bash
-  kubectl get svc carto-selfhosted-v1-router -o jsonpath='{.status.loadBalancer.ingress.*.ip}'
-  ```
-  
-  In EKS:
-
-  Create a CNAME record poiting to:
-
+5. Install your deployment:
   ```bash
-  kubectl get svc carto-selfhosted-v1-router -o jsonpath='{.status.loadBalancer.ingress.*.hostname}'
+  helm install \
+    <your_own_installation_name|carto> \
```

Comment:
I would put a default name, so all commands in the README are using the same name. If someone wants to change it to match some naming policy or whatever, they probably know how to do it. But with defaults we make the copy/paste easier

Reply:
and I don't like the default name because it is indistinguishable from the other parameters. Maybe use a BASH variable with a default value?